scripts/runner/setup

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
I've mode some minor updates to that in my other PR to rename `testnet` to `testnet-kintsugi`. Needs to be reflected here as well.

Also, can we not merge the two setup scripts together in one?

You could make a second arg so that it would be `setup testnet-kintsugi runner` or `setup testnet-interlay vault`...

Reply:
Done, but heads up that this will cause a breaking change with the existing setup (run `./setup kintsugi vault` instead of `./setup kintsugi`).